src/helpers/figures/charts/scorecard_chart.ts

```diff
@@ -347,16 +350,14 @@ export function drawScoreChart(structure: ScorecardChartConfig, canvas: HTMLCanv
   }
 
   if (structure.baselineDescr) {
-    const descr = structure.baselineDescr[0];
+    const descr = structure.baselineDescr[0]!;
     ctx.font = descr.style.font;
     ctx.fillStyle = descr.style.color;
-    for (const description of structure.baselineDescr) {
```

Comment:
the for loop is there because the baseline description can be split in two lines if if's too long. So I think this change breaks the behavior :sweat_smile: 

EDIT: it seems the mutli-line behavior has been removed already. It should no longer be an array then, but that's for a future refactoring